### ISSUE TITLE
TypeError: Cannot read property 'asyncOnly' of undefined

### ISSUE THREAD

**Issue by theodorDiaconu** [meteortesting/meteor-mocha#75]
```
I20180820-11:47:59.354(3)?   1) mocha-test
I20180820-11:47:59.355(3)?        server is not client:
I20180820-11:47:59.355(3)?      TypeError: Cannot read property 'asyncOnly' of undefined
I20180820-11:47:59.355(3)?       at run (packages/meteortesting:mocha-core/server.js:48:24)
I20180820-11:47:59.355(3)?       at Test.wrappedFunction (packages/meteortesting:mocha-core/server.js:63:33)
I20180820-11:47:59.355(3)?       at run (packages/meteortesting:mocha-core/server.js:56:16)```

This happens with meteor-mocha-core@5.2.0_1. Client tests work.

**Comment by theodorDiaconu** [comment]
This only happens for tests that use `it.only()`

**Comment by SimonSimCity** [comment]
I'll close this here as it should be fixed by updating to version `5.2.0_2` of the package `meteortesting:mocha-core`.

**Comment by hexsprite** [comment]
Fixed for me! Thanks @SimonSimCity, you rock!
